pkg/operator/controller/cruft.go

```diff
@@ -263,3 +268,93 @@ func reconcileRemainingRelationshipLabels(args *callbacks.ReconcileCallbackArgs)
 
 	return nil
 }
+
+// Delete after we no longer want to include CDI CRD v1alpha1 version in release YAMLs
+// Special code needed because we're not the owner of this object.
+func (r *ReconcileCDI) watchCDICRD() error {
+	if err := r.controller.Watch(&source.Kind{Type: &extv1.CustomResourceDefinition{}}, handler.EnqueueRequestsFromMapFunc(
+		func(obj client.Object) []reconcile.Request {
+			name := obj.GetName()
+			if name != "cdis.cdi.kubevirt.io" {
+				return nil
+			}
+			cr, err := cc.GetActiveCDI(r.client)
+			if err != nil {
+				return nil
+			}
+			return []reconcile.Request{
+				{
+					NamespacedName: types.NamespacedName{
+						Namespace: "",
+						Name:      cr.Name,
+					},
+				},
+			}
+		},
+	)); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// ReconcileCDICRD monitors the CDI CRD and removes the alpha version from it.
+// Delete after we no longer want to include CDI CRD v1alpha1 version in release YAMLs
+// Remove alpha as a version from the CDI CRD
+func reconcileCDICRD(args *callbacks.ReconcileCallbackArgs) error {
+	if args.State != callbacks.ReconcileStatePostRead {
+		return nil
+	}
+	deployment := args.CurrentObject.(*appsv1.Deployment)
+	if !isControllerDeployment(deployment) || !sdk.CheckDeploymentReady(deployment) {
```

Comment:
not a big deal, but don't think you have to wait for the deployment to be ready

Reply:
Done